docs/03-tutorials/00-eventing/evnt-04-change-max-in-flight-in-sub.md

```diff
@@ -91,16 +91,9 @@ Create a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01
      config:
        maxInFlightMessages: 5
      sink: 'http://lastorder.default.svc.cluster.local'
-     filter:
-       filters:
-         - eventSource:
-             property: source
-             type: exact
-             value: ''
-           eventType:
-             property: type
-             type: exact
-             value: sap.kyma.custom.myapp.order.received.v1
+     source: myapp
```

Comment:
From here for all the other occurences, please change the `apiVersion` to `eventing.kyma-project.io/v1alpha2`.

Reply:
Updated. Good catch (y)